code/game/gamemodes/objective_items.dm

```diff
@@ -65,12 +65,6 @@
 	difficulty = 3
 	excludefromjob = list("Head of Security", "Warden")
 
-/datum/objective_item/steal/reactive
```

Comment:
Is removing it from objectives necessary, considering it's still researchable and obtainable?